### ISSUE TITLE
总损失下降，map值先升后降

### ISSUE THREAD

**Issue by Weiziwei521** [JDAI-CV/fast-reid#263]
训练的时候总损失一直在下降，但是map值在90个epoch时79.3,120个epoch时78.1了，是什么原因呢?用的1个GPU，下面是训练生成的config.yaml
CUDNN_BENCHMARK: true
DATALOADER: {NAIVE_WAY: true, NUM_INSTANCE: 16, NUM_WORKERS: 8, PK_SAMPLER: true}
DATASETS:
  COMBINEALL: false
  NAMES: [DukeMTMC]
  TESTS: [DukeMTMC]
INPUT:
  DO_AUGMIX: false
  DO_AUTOAUG: false
  DO_CJ: false
  DO_FLIP: true
  DO_PAD: true
  FLIP_PROB: 0.5
  PADDING: 10
  PADDING_MODE: constant
  REA:
    ENABLED: true
    MEAN: [123.675, 116.28, 103.53]
    PROB: 0.5
  RPT: {ENABLED: false, PROB: 0.5}
  SIZE_TEST: [384, 128]
  SIZE_TRAIN: [384, 128]
MODEL:
  BACKBONE: {DEPTH: 50, LAST_STRIDE: 1, NAME: build_resnet_backbone, NORM: BN, NORM_SPLIT: 1,
    PRETRAIN: true, PRETRAIN_PATH: '', WITH_IBN: false, WITH_NL: false, WITH_SE: true}
  DEVICE: cuda:0
  FREEZE_LAYERS: ['']
  HEADS: {CLS_LAYER: circle, IN_FEAT: 2048, MARGIN: 0.15, NAME: BNneckHead, NECK_FEAT: before,
    NORM: BN, NORM_SPLIT: 1, NUM_CLASSES: 0, POOL_LAYER: gempool, REDUCTION_DIM: 512,
    SCALE: 128}
  LOSSES:
    CE: {ALPHA: 0.2, EPSILON: 0.1, SCALE: 1.0}
    CIRCLE: {ALPHA: 128, MARGIN: 0.25, SCALE: 1.0}
    FL: {ALPHA: 0.25, GAMMA: 2, SCALE: 1.0}
    NAME: [CrossEntropyLoss, TripletLoss]
    TRI: {HARD_MINING: false, MARGIN: 0.0, NORM_FEAT: false, SCALE: 1.0}
  META_ARCHITECTURE: Baseline
  PIXEL_MEAN: [123.675, 116.28, 103.53]
  PIXEL_STD: [58.395, 57.120000000000005, 57.375]
  WEIGHTS: ''
OUTPUT_DIR: logs/dukemtmc/agw_R50
SOLVER:
  BASE_LR: 0.01
  BIAS_LR_FACTOR: 2.0
  CHECKPOINT_PERIOD: 60
  DELAY_ITERS: 0
  ETA_MIN_LR: 3.0e-07
  FREEZE_ITERS: 0
  GAMMA: 0.1
  HEADS_LR_FACTOR: 1.0
  IMS_PER_BATCH: 64
  MAX_ITER: 120
  MOMENTUM: 0.9
  OPT: SGD
  SCHED: WarmupMultiStepLR
  STEPS: [40, 90]
  SWA: {ENABLED: false, ETA_MIN_LR: 3.5e-06, ITER: 10, LR_FACTOR: 10.0, LR_SCHED: false,
    PERIOD: 2}
  WARMUP_FACTOR: 0.01
  WARMUP_ITERS: 10
  WARMUP_METHOD: linear
  WEIGHT_DECAY: 0.0005
  WEIGHT_DECAY_BIAS: 0.0005
TEST:
  AQE: {ALPHA: 3.0, ENABLED: false, QE_K: 5, QE_TIME: 1}
  EVAL_PERIOD: 30
  IMS_PER_BATCH: 128
  METRIC: cosine
  PRECISE_BN: {DATASET: Market1501, ENABLED: false, NUM_ITER: 300}
  RERANK: {ENABLED: false, K1: 20, K2: 6, LAMBDA: 0.3}

**Comment by L1aoXingyu** [comment]
train 多了就 overfitting 了，很正常。

**Comment by KRYSTALLJY** [comment]
我也有相似的情况，可是如果只训练120epoch左右精度不够，那要如何不过拟合同时提高精度呢？
